### DISCUSSION TITLE
Rom Import Process Enhancement?

### DISCUSSION THREAD

**Discussion by BillDStrong** [retronas/discussions#17] [Q&A]
Am I wrong in the assumption the Rom Import process is reading the roms completely for each system, rather than saving the checksums and just comparing against the canonical files?

This is not good for older servers like mine that now must read in large file sets. It is a waste of computation and time. I don't know if this is a problem of the original importer or the way you are using it, but I would expect the roms to be checked once per run of the importer, and then compared through each of the systems.

**Comment by sairuk** [comment]
The behaviour you referring to is part of the main project, we only tell it to scan a known directory with a hash list and then put the files in the dest.

alternatively SabreTools is available for more advanced scanning
